### ISSUE TITLE
Bug report switching versions

### ISSUE THREAD

**Issue by SocksM** [5zig-reborn/The-5zig-Mod#84]
**Describe the bug**
So bassically when i switch versions it tends to delete my hud (like armout status etc)

**To Reproduce**
1. load up 5zig play on 1.8.9
2. load a higher mc version (i did it without any mods)
3. go load up 5zig on 1.8.9 again and see your hud disappeared

**Expected behavior**
Still have the hud when i switch back

**Screenshots**

**Desktop (please complete the following information):**
 - OS: windows 10 64bit
 - Version latest?
 - If built from source: ?

**Additional context**

**Comment by RoccoDev** [comment]
Yeah, this is an issue I've seen people have on the old mod as well. Can you send your Minecraft latest.log file before and after you switch versions?
